src/express-tooling.ts

```diff
@@ -199,12 +199,15 @@ const authenticationMiddleware: authenticationMiddleware = (options) => {
       notAuthenticatedHandler(res, logOrNothing, HttpStatus.UNAUTHORIZED);
       return;
     } else {
-      getTokenInfo(tokenInfoEndpoint, accessToken)
+      getTokenInfo(tokenInfoEndpoint, accessToken, logger)
         .then(setTokeninfo(req))
         .then(next)
         // TODO we should send 500 for issues with network etc.
         //      we should send HttpStatus.UNAUTHORIZED for invalid token
-        .catch(err => notAuthenticatedHandler(res, logOrNothing, HttpStatus.UNAUTHORIZED));
+        .catch(err => {
+          logOrNothing.warn(`Error while getting token info: ${err}`);
```

Comment:
If `err` is an object this will yield `"[object Object]"`.

I would suggest:

```ts
logOrNothing.warn(`Error while getting token info`, err);
```

...because the logger accepts an error object as second parameter which is stringified reasonably.

Reply:
Ah, makes sense. Now I noticed the logger's second argument was `any`. Pushed another commit to fix this and another line that was using a template literal.